src/components/PageTitle/PageTitle.scss

```diff
@@ -0,0 +1,24 @@
+@import "src/styles/font-styles";
+
+.page-title {
+  @extend .title;
```

Comment:
it's really not recommended to use @extend 
use a sass placeholder` %`  or a mixin instead.

https://www.sitepoint.com/avoid-sass-extend/
https://tiffanybbrown.com/2017/03/dont-use-extend-sass/index.html